Makefile

```diff
@@ -75,20 +75,22 @@ reqs_dev: requirements/requirements_dev.txt
 $(env_tag): requirements/requirements.txt
 	@echo "==Installing requirements.txt=="
 	pip-sync --quiet requirements/requirements.txt
-	rm -f $(env_dev_tag)
-	rm -f $(install_tag)
+	rm $(install_tag)
```

Comment:
Watch out that this change breaks the workflow since `rm $(install_tag)` throws an error in case `$(install_tag)` is missing (and that is the case when you set up the environment the first time). 
I think you should use the forced version `rm -f $(install_tag)`

Reply:
Yes, totally. Thanks